app/master/build.py

```diff
@@ -87,7 +101,7 @@ def prepare(self, subjobs, project_type, job_config):
 
         self._max_executors = job_config.max_executors
         self._max_executors_per_slave = job_config.max_executors_per_slave
-        self._timing_file_path = project_type.timing_file_path(job_config.name)
+        self._timing_file_path = self.project_type.timing_file_path(job_config.name)
```

Comment:
The project_type could be None at this point.  
Previously, this possibly was avoided by explicitly requiring the project_type be passed in as a parameter.

Should there be additional error handling logic here?